node/src/components/chain_synchronizer/operations.rs

```diff
@@ -863,7 +863,7 @@ async fn fast_sync(ctx: &ChainSyncContext<'_>) -> Result<(KeyBlockInfo, BlockHea
     sync_trie_store(&most_recent_block_header, ctx).await?;
 
     ctx.effect_builder
-        .update_lowest_available_block_height_in_storage(most_recent_block_header.height())
+        .mark_block_completed(most_recent_block_header.height())
         .await;
```

Comment:
I think this should be removed, as we haven't fetched the deploys for this block at this stage.

Reply:
Hmm, we really don't have any complete block after fast syncing, do we?

Reply:
No, we don't